solc/CommandLineInterface.cpp

```diff
@@ -917,6 +917,7 @@ bool CommandLineInterface::processInput()
 	else
 		formatter = make_unique<SourceReferenceFormatter>(serr(false));
 
+	bool errorRecovery = m_args.count(g_argErrorRecovery);
```

Comment:
I don't think this variable with such a long visibility is needed - you can just call the function twice.

Reply:
Not sure I understand the concern here, but this is trivial to change and is done in 023ebf659